crates/cli/src/config.rs

```diff
@@ -92,28 +92,30 @@ fn find_util_rules(
     return Ok(GlobalRules::default());
   };
   let mut utils = vec![];
-  // TODO: use WalkBuilder::add to avoid loop
+  let mut walker = WalkBuilder::new(&base_dir);
```

Comment:
I don't think walker should iterate through base_dir by default

Reply:
Also by default WalkBuilder::new(base_path) will add base_path to the iteration procedure. I think we should exclude it.